assets/js/atomic/components/product-list/image/index.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { Fragment } from 'react';
+import { __ } from '@wordpress/i18n';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { PLACEHOLDER_IMG_SRC } from '@woocommerce/block-settings';
+
+const ProductListImage = ( {
```

Comment:
In slow networks, there is a content jump while loading images:

![Peek 2019-10-09 11-50](https://user-images.githubusercontent.com/3616980/66472623-eaaceb00-ea8d-11e9-988d-57def07829b7.gif)

That's why in `product-grid-item/product-image/index.js` there was a `onImageLoaded` logic to prevent this kind of jumps. Do you think we could get that into the new atomic component?